{"id": "121687", "title": "Bug 121687: Exception thrown when exporting data in table which is inside a grid cell", "description": "", "OB": "When attempting to export data in a table that is inside a grid cell, an exception is thrown.", "EB": "When exporting data in two tables, the expected result is that the data should be exported. However, the actual result is that an exception occurs in the ReportEngineService.getMetaData() method.", "SR": "To reproduce this issue: \n1. Open the application\n2. Navigate to the page containing the grid with the table inside a cell\n3. Export the data in the table\n4. Observe the exception being thrown"}